src/mixins/accountMixins.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * @copyright Copyright (c) 2019 Cyrille Bollu <cyrpub@bollu.be>
+ *
+ * @author Cyrille Bollu <cyrpub@bollu.be>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/*
+ * This file provides global account related methods
+ */
+
+import serverData from './serverData'
+
+export default {
+	mixins: [
+		serverData
+	],
+	methods: {
```

Comment:
Please use a computed property to make sure it is properly updated if needed

Reply:
ok